devito/passes/clusters/cse.py

```diff
@@ -111,7 +112,10 @@ def _cse(maybe_exprs, make, min_cost=1, mode='default'):
         updated = []
         for e in processed:
             pe = e
-            for k, v in chosen:
+            pe_c = e.conditionals
```

Comment:
`pe` changes, `pe_c` doesn't, so you don't need an assignment here

following on from the other comments below, here I'd have:

```
for k, v in chosen:
    if k.conditionals != e.conditionals:
         continue
    ...
```

Reply:
> , so you don't need an assignment here

Just habit to assign if resused but yes not really necessary